packages/logger/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@loopback/logger",
+  "version": "1.0.0",
```

Comment:
Let's reserve `1.0.0` version for GA please. I am proposing to use `0.1.0` instead.